src/app/action/action.component.html

```diff
@@ -23,7 +23,8 @@
      *ngIf="config.moreActions?.length > 0">
   <button class="btn btn-link dropdown-toggle" type="button" dropdownToggle
           [ngClass]="{'disabled': config.moreActionsDisabled}"
-          (click)="initMoreActionsDropup($event)">
+          (click)="initMoreActionsDropup($event)"
+          aria-label="More Actions">
```

Comment:
Adding a default string is fine, but we shouldn't hard code it. This text could be very specific depending on the application.

We should allow this to be set via an ariaLabel property in the Action object used by ActionConfig. (Unless using the Action tooltip is sufficient?) As a default value, we can use the tooltip property.

Reply:
By Action object, I assume you're talking about actionConfig in the toolbar example component?  I added ariaLabel as an optional property on ActionConfig and set the value to "More Actions" if ariaLabel isn't specified. There wasn't already a tooltip property in ActionConfig so I didn't see sense in adding one just to set it as a default for ariaLabel, since the tooltip would be optional also.

The tooltip for the individual actions is a different thing though, since they are specific to each action. The ariaLabel here is really describing that entire "More Actions" kabab as a whole so it seems like adding a dedicated property for this, with a text fallback, is a good thing to do.

Thoughts?